libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt

```diff
@@ -21,6 +22,8 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
         navigator.updateLocation(rawLocation.toFixLocation())
     }
 
+    override fun getRoute(url: String): RouterResult = navigator.getRoute(url)
```

Comment:
We're adding synchronization in the call-site 👀 https://github.com/mapbox/mapbox-navigation-android/blob/b1e01add2ce335a55a5ddd47790fc0483a5b00a1/libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/task/OfflineRouteRetrievalTask.kt#L36 Should we remove it from `OfflineRouteRetrievalTask` and add it here?